Directory.Build.props

```diff
@@ -4,7 +4,6 @@
     <RepoEngPath>$(MSBuildThisFileDirectory)eng</RepoEngPath>
     <RepoSrcPath>$(RepoRoot)src</RepoSrcPath>
     <IsTestProject Condition="$(MSBuildProjectName.EndsWith('.Tests'))">true</IsTestProject>
-    <RestoreAdditionalProjectSources>https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-net/index.json</RestoreAdditionalProjectSources>
```

Comment:
This source should be added to nuget.config

Reply:
Isn't it the same as `<add key="azure-sdk-tools" value="https://azuresdkartifacts.blob.core.windows.net/azure-sdk-tools/index.json" />`?

Reply:
I accidentally substituted this with `https://azuresdkartifacts.blob.core.windows.net/azure-sdk-tools/index.json`.

Reply:
Yeah, I accidentally substituted this with `https://azuresdkartifacts.blob.core.windows.net/azure-sdk-tools/index.json`.

Added this to NuGet.Config.

Reply:
Yeah, my bad I accidentally substituted this with `https://azuresdkartifacts.blob.core.windows.net/azure-sdk-tools/index.json`.

Added this to NuGet.Config.